src/site/xdoc/index.xml

```diff
@@ -36,18 +36,20 @@
       <subsection name="Usage">
         <p>General instructions on how to use the Flatten Maven Plugin can be found on the <a href="usage.html">usage page</a>.</p>
         
-        <p>If you have questions regarding the plugin's usage, contact the <a href="mail-lists.html">user mailing list</a>. Posts to the mailing list are archived and could
+        <p>If you have questions regarding the plugin's usage, contact the <a href="https://www.mojohaus.org/flatten-maven-plugin/mailing-lists.html">user mailing list</a>.
+            Posts to the mailing list are archived and could
             already contain the answer to your question as part of an older thread. Hence, it is also worth browsing/searching
-            the <a href="mail-lists.html">mail archive</a>.
+            the mail archive.
         </p>
 
        <p>You can fill a feature request or bug report in our
-          <a href="issue-tracking.html">issue tracker</a>. When creating a new issue, please provide a comprehensive description of your
+          <a href="https://github.com/mojohaus/flatten-maven-plugin/issues">issue tracker</a>.
```

Comment:
We still have a local report about issue tracking system, I would point url to it.

Reply:
Updated to issue-management.html.